jenkins/scripts/open_search_db.py

```
@@ -266,71 +287,117 @@ def postToOpenSearchDB(json_data, project) -> bool:
+                            f"accepted with status {status_code}")
                     return True
-                else:
-                    OpenSearchDB.logger.info(
-                        f"OpenSearchDB post failed, will retry, error:{status_code} {last_text}"
-                    )
+
+                last_error = _S(f"HTTP {status_code}: {last_text[:200]}")
+                OpenSearchDB.logger.warning(
+                    f"{_T}[WARN](op=post, cat=transient, status={status_code}, db={project}, attempt={attempt}/{DEFAULT_RETRY_COUNT}) "
+                    f"will retry")
             except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)